### ISSUE TITLE
V2 class for errors has changed, please fix doc

### ISSUE THREAD

**Issue by SpencerBrown** [aws/aws-sdk-ruby#710]
Somewhere recently the base class for raised errors changed, I've fixed my code but the doc needs to be updated.

It is now just Aws::EC2::Errors instead of Aws::EC2::Errors::ServiceError.

**Comment by trevorrowe** [comment]
I am unable to reproduce this. I sent an invalid request, rescuing the raised error. Here are its ancestors:

```ruby
error.class.ancestors
=> [Aws::EC2::Errors::InvalidInstanceIDMalformed,
 Aws::EC2::Errors::ServiceError,
 Aws::Errors::ServiceError,
 RuntimeError,
 StandardError,
 Exception,
 Object,
 JSON::Ext::Generator::GeneratorMethods::Object,
 PP::ObjectMixin,
 Kernel,
 BasicObject]
```

You should be able to rescue Aws::EC2::Errors::ServiceError or the even more generic Aws::Errors::ServiceError.

**Comment by SpencerBrown** [comment]
That is strange. A few days ago, my code broke because it was rescuing
exceptions on Aws::EC2::Errors::ServiceError, and the SDK raised an
exception due to a Network Instance not existing, and my rescue didn't
catch it. This was repeatable. I changed my code to rescue
Aws::EC2::Errors, and it immediately started working again.

Now that I try to reproduce it, I'm not seeing that at all.

There may have been a fix in the latest SDK which I may have updated to.

Anyway, thanks for looking into this so quickly.

BTW which repo should I report issues on, now?  It wasn't clear so I opened
an issue on both.
